t/25-cache-service.t

```diff
@@ -104,7 +104,7 @@ sub test_sync {
     is $status->result, 'exit code 0', "exit code ok, run $run";
     ok $status->output, "output ok, run $run";
 
-    like $status->output, qr/Calling: rsync .* --timeout 1800 .*100\%/s, "output correct, run $run"
+    like $status->output, qr/Calling try \d: rsync .* --timeout 1800 .*100\%/s, "output correct, run $run"
```

Comment:
This does not actually test that it tried more than once.

Reply:
Yes, and it would be nice to have a test.